### PR TITLE
Add test data

### PR DESCRIPTION
**Pull request by lfdgov**
# Description

This PR includes the following proposed change(s):

- adds test data for JJ workbench
- adds missing env variables to template
- adds missing ticket search type to docker compose
- update generated files through to client

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

tested JJ workbench functional

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Sharkww — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ services:
       REDIS__CONNECTIONSTRING: ${REDIS__CONNECTIONSTRING:-redis:6379,password=password}
       TICKETSEARCH__SEARCHTYPE: ${TICKETSEARCH__SEARCHTYPE:-Mock}
       TICKETSTORAGE: ${TICKETSTORAGE:-ObjectStore}
+      TICKETSEARCH__SEARCHTYPE: ${TICKETSEARCH__SEARCHTYPE:-Mock}
```
Comment (KDNeufeld) [inline docker-compose.yml:32]: This is a duplicate entry. See line 30.

File: src/backend/oracle-data-api/src/main/resources/data.sql [hunk 2]
```diff
@@ -25,8 +25,15 @@ INSERT INTO JJDISPUTE (TICKET_NUMBER, CREATED_BY, CREATED_TS, MODIFIED_BY, MODIF
...
+  ('AO38375806', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Charles', 'LaRoque', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', 'French', 10, 'IN_PERSON'),
+  ('AU46117416', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Rebecca', 'Klein', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', '', '', '', '', '', '', 3, 'IN_PERSON'),
+  ('ST04605408', 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'Robson Square Provincial Court', 'Brian', 'McIntosh', 'Victoria', '', '', 'Steven Allan', 'colmohig', 'West Shore', LOCALTIMESTAMP, 'HEARING_SCHEDULED', LOCALTIMESTAMP, LOCALTIMESTAMP, 'COURT_APPEARANCE', 'Lional Hutz Storage Facility', 'Hutz', 'Lionel', '', '', '', 0, 'IN_PERSON');
+
```
Comment (KDNeufeld) [inline src/backend/oracle-data-api/src/main/resources/data.sql:36]: These ticket numbers don't match anything in our mock data (test-ticket-data.csv used by the ticket search).  This means that we have Disputes without matching JJDisputes and visa versa.

Comment (lfdgov) [reply src/backend/oracle-data-api/src/main/resources/data.sql:36]: These were the ones requested by business.

File: src/backend/oracle-data-api/src/main/resources/data.sql [hunk 3]
```diff
@@ -70,26 +84,60 @@ INSERT INTO JJDISPUTED_COUNT (ID, CREATED_BY, CREATED_TS, MODIFIED_BY, MODIFIED_
...
+(30, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA00241528', 'EA00241528', 3, 'Y', '', '92.1(1) MVA - Fail to stop resulting in pursuit', 1, 'N', 380.00, 380.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(31, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA07789632', 'EA07789632', 1, 'Y', '', '146(1) Speed in/outside Municipality', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(32, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA01115462', 'EA01115462', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(33, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 196.00, 196.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(34, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 2, 'Y', '', '214.2(1) Using Electronic device while driving', 0, 'N', 368.00, 368.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
```
Comment (KDNeufeld) [inline src/backend/oracle-data-api/src/main/resources/data.sql:101]: This statute "214.2(1)" is not in the current list of available statutes from JUSTIN.

File: docker-compose.yml (hunk 1 above)
Comment (lfdgov) [reply docker-compose.yml]: Corrected -- thanks

File: src/backend/oracle-data-api/src/main/resources/data.sql [hunk 4]
```diff
@@ -70,26 +84,60 @@ INSERT INTO JJDISPUTED_COUNT (ID, CREATED_BY, CREATED_TS, MODIFIED_BY, MODIFIED_
...
+(35, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA04788953', 'EA04788953', 3, 'Y', '', '214.2(2) Emailing or texting while driving', 1, 'N', 368.00, 368.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(36, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'EA05888444', 'EA05888444', 1, 'Y', '', '146(3) Speed against highway sign', 0, 'N', 138.00, 138.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
+(37, 'System', LOCALTIMESTAMP, 'System', LOCALTIMESTAMP, 'AO38375806', 'AO38375806', 1, 'Y', '', '194(3) Ride motorcycle without required helmet', 1, 'N', 138.00, 138.00, 'Y', 'N', DATEADD('DAY', 30, CURRENT_DATE), null),
```
Comment (KDNeufeld) [inline src/backend/oracle-data-api/src/main/resources/data.sql:104]: It looks like the statutes are different in this data than what's in JUSTIN. This id "194(3)" refers to "Permit unlawfully seated passenger", not "Ride motorcycle without required helmet"

**Review by KDNeufeld — APPROVED** [review]

File: src/backend/oracle-data-api/src/main/resources/data.sql (hunk 3 above)
Comment (lfdgov) [reply src/backend/oracle-data-api/src/main/resources/data.sql:101]: It is possible that an officer may write an incorrect statute down on a ticket, so this is still ok test data -- and its what was requested by business.

**Pull request merged**